index.html

```diff
@@ -0,0 +1,625 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The New York Times Article Clone</title>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@300;400&family=Noto+Serif:ital@1&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="assets/normalize.css" type="text/css">
+    <link rel="stylesheet" href="assets/style.css" type="text/css">
+    <script src="https://use.fontawesome.com/aaef801754.js"></script>
+    <script src="https://kit.fontawesome.com/d86848cfe0.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav-bar flex">
+            <ul class="hamburger-search flex">
+                <li><i class="fa fa-bars" aria-hidden="true"></i></li>
+                <li><i class="fa fa-search" aria-hidden="true"></i></li>
+            </ul>
+            <ul class="nav-logo">
+                <li><a><img src="assets/images/logo.png" alt="New York Times Logo"></a></li>
+            </ul>
```

Comment:
## The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

- [ ] Your're missing the default link, please add `<a href="#">`